node/test/v2/reader.js

```diff
@@ -21,9 +21,10 @@
 'use strict';
 
 var util = require('util');
-var Parser = require('../../v2/parser.js');
+var Reader = require('../../v2/reader.js');
 var TestFrame = require('./test_frame');
-var parserTest = require('../lib/parser_test');
```

Comment:
it got renamed to `test/lib/test_expectations`, I already had the primitive factored out internally, just not exposed yet.